### ISSUE TITLE
Cannot subtract days from CURRENT_DATE

### ISSUE THREAD

**Issue by JanisOzolins** [oguimbal/pg-mem#172]
# Describe the bug

I have a Postgres query where I am trying to select a list of record with a timestamp within the last 7 days.

```
error properties: Object({ data: Object({ error: 'cannot cast type integer to date', code: undefined }), location: Object({ start: 0, end: 0 }) })
```

If I remove ` - 7` from the query it works. Is there a way or workaround to subtract 7 days from the CURRENT_DATE to achieve the same result?

# To Reproduce

```sql

create schema prod_ui;
      create table prod_ui.users_active(project_id text, user_id text, last_Active DATE);
      insert into prod_ui.users_active values ('project1', 'user1', '2021-11-01 15:36:45');
      insert into prod_ui.users_active values ('project1', 'user2', '2021-11-01 15:36:45');

SELECT user_id FROM prod_ui.users_active ua
  WHERE ua.project_id='project1'
  AND ua.last_Active > (CURRENT_DATE - 7)
  GROUP BY user_id
      

```

# pg-mem version

2.1.9

**Comment by oguimbal** [comment]
Thanks for the feedback. Will fix.

Workaround: Use the explicit notation for intervals:  `CURRENT_DATE -  interval '7 days'` 

Note to myself or to whever wants to fix this: it should just be an implicit cast number -> timespan which is missing.

**Comment by JanisOzolins** [comment]
Thank you, indeed that is a workaround for now.

**Comment by oguimbal** [comment]
Hi, this should be fixed in `pg-mem@2.2.0` (reopen if not !)

Sorry for the delay !
